common/src/nostr/mod.rs

```diff
@@ -48,14 +48,15 @@ impl NostrClient {
         None
     }
 
+    // coordinator specific function?
```

Comment:
I think we can put these functions in the client as they are not used by the coordinator